Makefile

```
@@ -35,3 +35,7 @@ $(plugins_dir):
 
 test:
 	go test ./...
+
+%/remove: beta-notice
+	$(eval plugin := $(firstword $(subst /, ,$@)))
+	rm -rf ~/.op/plugins/local/$(plugin)
```

Can leave out `-r` to avoid any surprises in case the commands ends up pointing to a directory for any reason?
```suggestion
	rm -f ~/.op/plugins/local/$(plugin)
```

---

My mistake. I thought it was a directory! Fixing.